src/api_sync.rs

```diff
@@ -464,7 +464,7 @@ impl ApiSync {
         }
     }
 
-    /// Checks for a MAGLAG error, and returns the lag if so
+    /// Checks for a MAXLAG error, and returns the lag if so
```

Comment:
I think this can be lowercase, like it is everywhere else.

Reply:
done https://github.com/magnusmanske/mediawiki_rust/compare/9cf2e4a7f19fc8840e8c609f93d04e36be504a02..f8782274ce1f4af55f5302e497b873d73d3884c7